.github/workflows/long-running-azure.yaml

```diff
@@ -186,15 +187,15 @@ jobs:
           * ucp test image location: `${{ env.CONTAINER_REGISTRY }}/ucpd:${{ steps.gen-id.outputs.REL_VERSION }}`
 
           </details>
-          
+
```

Comment:
removing space might ignore the line space. can you please check if removing space adds line between L189 and L191?